openassessment/data.py

```diff
@@ -498,6 +500,111 @@ def collect_ora2_data(cls, course_id):
         ]
         return header, rows
 
+    @classmethod
+    def collect_ora2_summary(cls, course_id):
+        """
+        Query database for aggregated ora2 summary data.
+
+        Args:
+            course_id (string) - the course id of the course whose data we would like to return
+
+        Returns:
+            A tuple containing two lists: headers and data.
+
+            headers is a list containing strings corresponding to the column headers of the data.
+            data is a list of lists, where each sub-list corresponds to a row in the table of all the data
+                for this course.
```

Comment:
Could you provide a full description of each of the columns returned by this method, and the expected values?  

E.g., the `num_peers_graded` and `num_graded_by_peers` is a little hard to distinguish just from the header name, so would be good to have a text description of what the difference is.